src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -115,8 +115,8 @@ public function __invoke(Request $request): JsonResponse
                 foreach ($property as $prop) {
                     if (!$datagrid->hasFilter($prop)) {
                         throw new \RuntimeException(sprintf(
-                            'To retrieve autocomplete items,'
-                            .' you should add filter "%s" to "%s" in configureDatagridFilters() method.',
+                            'To retrieve autocomplete items,'.
+                            ' you should add filter "%s" to "%s" in configureDatagridFilters() method.',
```

Comment:
The concat operator was every time on the new line. Why change this ?

Reply:
For uniformity of code style.

Reply:
Best way would be to add an option to the `concat_space` rule of https://github.com/FriendsOfPHP/PHP-CS-Fixer to restrict the concat operator to the start of a new line, or the end of a new line.

When looking at my Sonata vendors, I find 114 concat at the start of the line ` .'` and only 63 at the end of the line `'.`.

If we start adding consistency, it should follow the majority IMHO.

Reply:
The ratio of 1 to 2 is not an indicator of the majority, but an indicator of the mess. I accept your opinion and redo it.